scala/ppml/demo/README.md

```diff
@@ -1,36 +1,63 @@
-# PPML Demo
+# Run PPML Demo in Graphene
 
-## Get Prepared
-### Get jar ready
-#### Build from source
+## Before running code
+
+### Build ppml-jar
+run:
 ```bash
-git clone https://github.com/intel-analytics/BigDL.git
-cd BigDL/scala
-./make-dist.sh
+cd .. && mvn clean package -DskipTests -Pspark_3.x
+mv target/bigdl-ppml-spark_3.1.2-0.14.0-SNAPSHOT-jar-with-dependencies.jar demo
+cd demo
 ```
-#### Download pre-build
+
+### Build Image
+Modify your http_proxy in `build-image.sh` then run:
 ```bash
-wget
+./build-image.sh
 ```
-### Config
-If deploying PPML on cluster, need to overwrite config `./ppml-conf.yaml`. Default config (localhost:8980) would be used if no `ppml-conf.yaml` exists in the directory.
-### Start FL Server
+
+### Prepare key and password
+Following these steps to prepare [key](http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-PPML-UTs-Validation/8/console) and [password](http://10.112.231.51:18888/view/ZOO-PR/job/BigDL-PR-PPML-UTs-Validation/8/console).Then modify these path and your local ip in `deploy-local-spark-sgx.sh`.
```

Comment:
This link/IP is internal only. Remove them from public README.

Reply:
Updated.